airbyte-integrations/bases/standard-destination-test/build.gradle

```diff
@@ -5,6 +5,9 @@ dependencies {
     implementation project(':airbyte-db:db-lib')
     implementation project(':airbyte-commons-worker')
     implementation project(':airbyte-config:config-models')
+    implementation project(':airbyte-config:config-persistence')
+    implementation project(':airbyte-config:init')
```

Comment:
Same comment as above, we are just running into some issues with the tests that are depending on the platform too strongly.

Reply:
I removed extra dependency `airbyte-config:config-persistence`. But I cann`t remove `airbyte-config:init` becouse I used `LocalDefinitionsProvider` in `DestinationAcceptanceTest.class`